### PR TITLE
{lang}[GCCcore/11.2.0] Python v3.9.6, Rust v1.54.0

### PR DESCRIPTION
**Pull request by SebastianAchilles**
(created using `eb --new-pr`)

Not the latest version in the EasyConfig for Python v3.9.6 are the extensions:
- `packaging`, because `poetry 1.1.7 has requirement packaging<21.0,>=20.4`,
- `keyring`, because `poetry 1.1.7 has requirement keyring<22.0.0,>=21.2.0`.

### CODE REVIEW

File: easybuild/easyconfigs/p/Python/Python-3.9.6-GCCcore-11.2.0-bare.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+dependencies = [
+    ('binutils', '2.37'),
+    ('bzip2', '1.0.8'),  # required for bz2 package in Python stdlib
+    ('zlib', '1.2.11'),
+    ('libreadline', '8.1'),
+    ('ncurses', '6.2'),
+    ('SQLite', '3.36'),
+    ('XZ', '5.2.5'),
+    ('libffi', '3.4.2'),
+    ('OpenSSL', '1.1', '', True),
```
Comment (Micket) [inline easybuild/easyconfigs/p/Python/Python-3.9.6-GCCcore-11.2.0-bare.eb:29]: I suspect we don't need all of these for the bare version? Or do we?

Comment (Micket) [reply easybuild/easyconfigs/p/Python/Python-3.9.6-GCCcore-11.2.0-bare.eb:29]: Ok looking for a build log for 3.9.5 it does seem to do have all that stuff int he base python. I didn't expect sqlite

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
jwvis00.juwels - Linux centos linux 8.3.2011, x86_64, Intel(R) Xeon(R) Gold 6148 CPU @ 2.40GHz, Python 3.6.8
See https://gist.github.com/b227e0a2d29a9538916fb85577f65488 for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
jrlogin11.jureca - Linux centos linux 8.3.2011, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/0b18bd2b105d16311dff9a81eb5a89a7 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 4 out of 4 (3 easyconfigs in total)
jsfl01.jusuf - Linux centos linux 7.9.2009, x86_64, AMD EPYC 7742 64-Core Processor, Python 3.6.8
See https://gist.github.com/67f48ea918c545ba8814442f7e221f97 for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
alvis-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 3.6.8
See https://gist.github.com/21e66cd1be390caa9d26691f87efe245 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u08b.bear.cluster - Linux Ubuntu 20.04, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.8.5
See https://gist.github.com/96ed9c4e3c9793a294acc0bd6bf029f7 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0211u03a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/c9827c3aa5797330a5033a81f34b0115 for a full test report.

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bear-pg0305u15a.bear.cluster - Linux RHEL 8.4, POWER, 8335-GTX (power9le), Python 3.6.8
See https://gist.github.com/b22637a8277138506c1b0ba2f2094263 for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @SebastianAchilles!

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3512.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/57c35c04f62590c4f85ed5f6c7310be4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 23 out of 23 (3 easyconfigs in total)
node2659.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/82937854eb54c6907d285623c96abeee for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
select-pika-c6gd-2xlarge-0001 - Linux centos linux 8.3.2011, AArch64, ARM UNKNOWN (graviton2), Python 3.6.8
See https://gist.github.com/e446782028cf2f762a84ef74d4221762 for a full test report.

**Comment by lexming** [comment]
@boegelbot please test @ generoso

**Comment by lexming** [comment]
@boegelbot: please test @ generoso
EB_ARGS=Python-3.9.6-GCCcore-11.2.0.eb

**Comment by lexming** [comment]
@boegelbot: please test @ generoso
EB_ARGS=Python-3.9.6-GCCcore-11.2.0.eb

**Pull request merged**